content/shmem_reductions.tex

```diff
@@ -213,7 +213,7 @@
     restored to the original values.
 
     The sum and product reduction routines include complex-typed interfaces
-    for the \Cstd API only.
+    for the \Cstd and \Fortran API only.
```

Comment:
Even though, this is a correct clarification. Do we have to change it, given that Fortran is deprecated and we stopped adding any Fortran related interfaces ?

Reply:
I think we need to add it. We're not adding any new Fortran interfaces in 1.4, however, this change is clarifying that the *existing* Fortran sum and product reductions on complex types are still available as they were in 1.3. Without this text, their existence would be ambiguous because they would be prototyped in the Fortran API declarations, but the text (before the change) would seem to preclude them.